changes/2852.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix default icon search in standalone/frozen macOS applications that are not built as .app bundles (e.g., POSIX build made with PyInstaller).
```

Comment:
Apps bundled as standalone frozen binaries (e.g., POSIX builds made with PyInstaller) no longer crash on startup when trying to resolve the app icon.